### ISSUE TITLE
Apply command does nothing when .. is selected

### ISSUE THREAD

**Issue by martinprikryl** [michaellukashov/Far-NetBox#194]
NetBox 2.2.1 build 430
FAR 3.0 build 4562 x86

- Start SFTP session
- Keep the `..` "directory" focused
- *NetBox commands > Apply command* (or <kbd>Ctrl+G</kbd>)
- Nothing happens.

Either the command should be disabled or the command should work nevertheless as you can use custom commands that do not rely on any files (while that is actually hardly relevant for FAR as you can use FAR's native functions to execute plain commands).

**Comment by VictorVG** [comment]
Yes, while CtrlG and can not work, then I confirm. But it is possible through the Far -> CtrlG enter the any command with a prefix netbox: -> Enter and after disclosure metacharacters Far will give it to the plug-in performance. 

I think so worth a try.

**Comment by VictorVG** [comment]
Better. When you enter a command fails not, but after clicking OK, there is AV. Dump I took off, I'll try to look to the PDB ...

![nb432](https://cloud.githubusercontent.com/assets/906886/13382155/5bfb08a2-de7e-11e5-977e-fa67db7aefcc.png)

![dbg](https://cloud.githubusercontent.com/assets/906886/13382158/63bf4ef4-de7e-11e5-8c4a-934b15305009.png)
![dbg2](https://cloud.githubusercontent.com/assets/906886/13382157/63be18ea-de7e-11e5-986e-151bf7bef344.png)
![dbg3](https://cloud.githubusercontent.com/assets/906886/13382162/63bf9030-de7e-11e5-9c16-f56947cba4b0.png)
![dbg4](https://cloud.githubusercontent.com/assets/906886/13382161/63bf7730-de7e-11e5-9243-6b154ce292af.png)
![dbg5](https://cloud.githubusercontent.com/assets/906886/13382159/63bf68d0-de7e-11e5-9d03-3d55db39ef8f.png)
![dbg6](https://cloud.githubusercontent.com/assets/906886/13382160/63bf7d02-de7e-11e5-9a35-711a4e2656fc.png)

**Comment by VictorVG** [comment]
Better. When you enter a command fails not, but after clicking OK, there is AV. Dump I took off, I'll try to look to the PDB ...

![nb432](https://cloud.githubusercontent.com/assets/906886/13382155/5bfb08a2-de7e-11e5-977e-fa67db7aefcc.png)

![dbg](https://cloud.githubusercontent.com/assets/906886/13382158/63bf4ef4-de7e-11e5-8c4a-934b15305009.png)
![dbg2](https://cloud.githubusercontent.com/assets/906886/13382157/63be18ea-de7e-11e5-986e-151bf7bef344.png)
![dbg3](https://cloud.githubusercontent.com/assets/906886/13382162/63bf9030-de7e-11e5-9c16-f56947cba4b0.png)
![dbg4](https://cloud.githubusercontent.com/assets/906886/13382161/63bf7730-de7e-11e5-9243-6b154ce292af.png)
![dbg5](https://cloud.githubusercontent.com/assets/906886/13382159/63bf68d0-de7e-11e5-9d03-3d55db39ef8f.png)
![dbg6](https://cloud.githubusercontent.com/assets/906886/13382160/63bf7d02-de7e-11e5-9a35-711a4e2656fc.png)

**Comment by VictorVG** [comment]
Test - go to ftp://ftp.pcbsd.org/pun/, select ".." on panel, CtrlG -> error messages:

![test](https://cloud.githubusercontent.com/assets/906886/13382202/b373c3a2-de7f-11e5-9b6f-640c6d03a0fb.png)

Ok, test success. This bug is fixed.

**Comment by VictorVG** [comment]
Interesting cinema - without NetBox.map / NetBox.pdb click YES - crash if they are - work great if they slip after the accident - the stack file names appear where there was a failure, well, that's understandable. Because thrust to the archive itself and test a bunch of x64 to make it easier to find where is still there?

All collected in MS VC ++ 2010.

![bug](https://cloud.githubusercontent.com/assets/906886/13382359/ca6b0bc0-de83-11e5-9444-813c2bbf49e2.png)

ftp://ftp.pcbsd.org/pub/system-updates/ - this is not real files - is cascades symlink (это не файлы а какскад симлинков с которыми пока только FileZilla или FTPRush без вопросов разбираются, а иные клиенты на них просто в обморок падают. Нашатыря на них нет.:))

![pcbsd](https://cloud.githubusercontent.com/assets/906886/13382397/c24dd228-de84-11e5-8aaf-21c5e512d603.png)

Dump's (no PDB and added PDB) test screensot, test kit - [**NB432_test.7z**](http://www86.zippyshare.com/v/OQQw06AF/file.html).

???????

**Comment by VictorVG** [comment]
This bug is fixed in NetBox v2.2.1.432. I test this - passed
